accord-core/src/test/java/accord/Utils.java

```diff
@@ -118,6 +123,16 @@ public static Txn writeTxn(Ranges ranges)
         return new Txn.InMemory(ranges, MockStore.read(ranges), MockStore.QUERY, MockStore.update(ranges));
     }
 
+    public static Txn listWriteTxn(Node.Id client, Keys keys)
+    {
+        ListUpdate update = new ListUpdate(Function.identity());
+        for (Key k : keys)
+            update.put(k, 1);
```

Comment:
this isn't 100% correct... if you run this txn twice the second one will freak out as ListRequest won't like this... it works just fine for single run cases (which is what I do)...  I guess I could add a `int...` or something to allow overriding... but didn't want to complicate things too much

Reply:
possible... I really didn't bother testing... I think `Timestamped` doesn't like this... its been awhile...

Reply:
that's fine. If it has sharp edges that make it unsuitable outside of a narrow use case though, it shouldn't go in Utils.

Reply:
ill test and push this to the test if it has such a sharp edge (think it does)

Reply:
tested, we do just append, so this is actually safe...